src/Components/AlbumCard/AlbumCard.css

```diff
@@ -65,12 +65,15 @@ color: var(--primary-color);
 }
 
 .card_genre {
-color: var(--text-color);
+color: var(--accent-glow);
 background-color: var(--secondary-color);
 /* padding: 1vh 2vw 1vh 2vw; */
-padding: 2%;
+padding: 5%;
 margin: 1%;
 font-weight: bolder;
+border-radius: 5px;
+background: radial-gradient(ellipse at 83% 0, var(--secondary-color) 45%, transparent 100%, var(--primary-color));
```

Comment:
The slight transparency is a little tough on the eyes. We may be better served with a more opaque option.

Reply:
I kept messing with these and after getting to a point of just changing numbers a bunch for 45 mins I then realized we have not chosen a background yet and I think adjusting all components together may be the most efficient. I have also run into a wall of being able to manipulate this so some paired programming would be appreciated. I don't like how it looks but this was the best I could do at this point

Reply:
I do like it, it is just not the best it could be

Reply:
Also all the text color is kind of hard to read, I am surprised it passed the contrast test, Need to review as a group

Reply:
I set it arbitrarily really so spending another session on styling direction sounds like a great idea.
Would say though that rather than you changing it now we can spend another branch to further consolidate styling and set a background image